README.md

```diff
@@ -64,21 +64,12 @@ released.**
 
 ### Initialization
 
-The pipeline must be initialized before use. Initialization will download and
-process data and setup computational environment for R and python.
-
-To initialize, enter the following in the command line:
+The repository must be cloned onto local machine before analyses can proceed:
 
 ```sh
-# Login to synapse to download controlled-access data
-synapse login
-
-# Create and activate conda environment
-conda env create --quiet --force --file environment.yml
-source activate pancancer-classifier
+git clone git@github.com:greenelab/pancancer.git
```

Comment:
Is this sufficient to get the data from git-lfs?

Reply:
I believe if the user has Git LFS installed, this command will download LFS assets.

Reply:
Yeah, I think that is how it would work too. I will add that note here as well